imperative/python/megengine/functional/math.py

```diff
@@ -1152,31 +1152,57 @@ def dot(inp1: Tensor, inp2: Tensor) -> Tensor:
 
 
 def svd(inp: Tensor, full_matrices=False, compute_uv=True) -> Tensor:
-    r"""Computes the singular value decompositions of input matrix.
-
-    Args:
-        inp: input matrix, must has shape `[..., M, N]`.
-
-    Returns:
-        output matrices, `(U, sigma, V)`.
-
-    Examples:
-
-        .. testcode::
-
-            import numpy as np
-            from megengine import tensor
-            import megengine.functional as F
-
-            x = tensor(np.arange(0, 6, dtype=np.float32).reshape(2,3))
-            _, y, _ = F.svd(x)
-            print(y.numpy().round(decimals=3))
-
-        Outputs:
-
-        .. testoutput::
-
-            [7.348 1.   ]
+    """
+    Singular Value Decomposition.
+    When `inp` is a 2D array, it is factorized as ``inp = (U * sigma) @ V``, 
+    where `U` and `V` are 2D unitary arrays and `sigma` is a 1D
+    array of `inp`'s singular values. When `inp` is higher-dimensional, SVD is
+    applied in stacked mode as explained below.
+
+    Parameters
+    ----------
+    inp : (..., M, N) array_like
+        A real array with ``inp.ndim >= 2``.
+    full_matrices : bool, optional
+        If False (default), `U` and `V` have the shapes  ``(..., M, K)`` 
+        and ``(..., K, N)``, respectively, where ``K = min(M, N)``. 
+        Otherwise, the shapes  are ``(..., M, M)`` and 
+        ``(..., N, N)``, respectively.  
+    compute_uv : bool, optional
+        Whether or not to compute `U` and `V` in addition to `sigma`.  True
+        by default.
+
+    Returns
+    -------
+    U : { (..., M, M), (..., M, K) } array
+        Unitary array(sigma). The first ``inp.ndim - 2`` dimensions have the same
+        size as those of the input `inp`. The size of the last two dimensions
+        depends on the value of `full_matrices`. Only returned when
+        `compute_uv` is True.
+sigma : (..., K) array
+        Vector(sigma) with the singular values, within each vector sorted in
+        descending order. The first ``inp.ndim - 2`` dimensions have the same
+        size as those of the input `inp`.
+    V : { (..., N, N), (..., K, N) } array
+        Unitary array(s). The first ``inp.ndim - 2`` dimensions have the same
+        size as those of the input `inp`. The size of the last two dimensions
+        depends on the value of `full_matrices`. Only returned when
+        `compute_uv` is True.
```

Comment:
`VH` might be better than `V` since the return `V` is the transpose of `v` in svd algorithm notation. Here `V` is still kept consistent with the implementation.  But sometimes it may mislead users.